docs-chef-io/content/server/upgrades.md

```diff
@@ -619,21 +619,75 @@ To upgrade to Chef Infra Server on a tiered Chef Infra Server configuration, do
 
 1. Back up the Chef Infra Server data before starting the upgrade process using [knife-ec-backup](https://github.com/chef/knife-ec-backup).
 
-2. Confirm that the Chef Infra Server services are operational:
+2. Chef Infra Server 14.14 supports external OpenSearch for indexing.
+
+#### Steps To Enable External OpenSearch
```

Comment:
Did we mean to add the steps for tiered setup?

Reply:
@PrajaktaPurohit As discussed, Now the chef server is with internal opensearch. If that's the case then we don't need to mention any steps explicitly for tiered setup.